rasa_core/policies/online_trainer.py

```diff
@@ -153,9 +155,14 @@ def probabilities_using_best_policy(self, tracker, domain):
     @staticmethod
     def _export_stories(tracker):
         # export current stories and quit
-        export_file_path = utils.request_input(
-                prompt="File to export to (if file exists, this "
-                       "will append the stories) [stories.md]: ")
+        file_promt = ("File to export to (if file exists, this "
```

Comment:
minor thing, but correctly spelled it's "file_prompt" 😄

Reply:
:D